crypto/include/internal/aria.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2006-2017 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/* ====================================================================
+ * Copyright (c) 2017 Oracle and/or its affiliates.  All rights reserved.
+ */
+
+#ifndef HEADER_ARIA_H
+# define HEADER_ARIA_H
+
+# include <openssl/opensslconf.h>
+
+# ifdef OPENSSL_NO_ARIA
+#  error ARIA is disabled.
+# endif
+
+# include <stddef.h>
+
+# define ARIA_ENCRYPT     1
+# define ARIA_DECRYPT     0
+
+# define ARIA_BLOCK_SIZE    16  /* Size of each encryption/decription block */
+# define ARIA_MAX_KEYS      17  /* Number of keys needed in the worst case  */
+
+# ifdef  __cplusplus
+extern "C" {
+# endif
+
+typedef unsigned char ARIA_u128[ARIA_BLOCK_SIZE];
+
+struct aria_key_st {
+    unsigned int rounds;
+    ARIA_u128 rd_key[ARIA_MAX_KEYS];
+};
+typedef struct aria_key_st ARIA_KEY;
+
+
+extern int aria_set_encrypt_key(const unsigned char *userKey, const int bits,
+                        ARIA_KEY *key);
```

Comment:
I don't follow why is it declared `extern`. It can only be styling question, but it's not our style. But our styling rules say that second line above should be aligned. And I suppose if you drop `extern` it will be aligned...